### PR TITLE
tc/data-access

### PR DESCRIPTION
**Pull request by CedThen**
added data-access file where i extracted out all the db.Whatever.action calls so they're more easily reused

### CODE REVIEW

**Comment by CedThen** [comment]
other one first

File: server/routes/schedules.js [hunk 1]
```diff
@@ -16,7 +15,7 @@ const router = express.Router();
 router.get('/schedules', certify, async (req, res, next) => {
   try {
     const { churchId } = req.query;
-    const schedules = await retrieveChurchSchedules(churchId);
+    const schedules = await findAllChurchSchedules(churchId);
```
Comment (showduhtung) [inline server/routes/schedules.js:18]: I'm assuming these are just naming changes? lgtm

File: server/services/dataAccess.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import db from '../index';
```
Comment (showduhtung) [inline server/services/dataAccess.js:1]: like a lot

**Comment by showduhtung** [comment]
looks like an abstraction pr right? looks real good to  e

**Pull request merged**